### DISCUSSION TITLE
Does anyone know how I get rid of the generation history bar at the bottom?

### DISCUSSION THREAD

**Discussion by lennysunreal** [Comfy-Org/ComfyUI#1599] [Q&A]
Hi hi;
Does anyone know how I get rid of the generation history bar at the bottom? 

![image](https://github.com/comfyanonymous/ComfyUI/assets/24582071/df2db03f-61f6-4661-a285-2057b78be87d)

**Comment by funkyman228** [comment]
I think there's a close button on the right, but you need to move your menu out of the way to see it.

**Reply by lennysunreal** [reply]
Sadly when I make the window wider it resizes and stays hidden. BUT good to know its likely under there. I'll try some different browsers.

**Reply by lennysunreal** [reply]
Ahhhh haaa on Brave Browse you can't get to the X. On Chrome I can

**Reply by brknsoul** [reply]
You could change the position to the side, that should show the X
